src/Chapter01/Listing01.09.RemovingWhitespace.cs

```diff
@@ -2,6 +2,6 @@ namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter01.Listing01_09
 {
     #region INCLUDE
     public class HelloWorld{public static void Main()
-{System.Console.WriteLine("Hello Inigo Montoya");}}
+{ Console.WriteLine("Hello Inigo Montoya");}}
```

Comment:
Inconsistent spacing and too many closing braces
```suggestion
{ Console.WriteLine("Hello Inigo Montoya"); }
```